src/schema/sources.ts

```diff
@@ -117,6 +118,11 @@ export const typeDefs = /* GraphQL */ `
     Number of members in the source
     """
     membersCount: Int!
+
+    """
+    Logged Member role
+    """
+    currentMember: String
```

Comment:
I think it would be more consistent to return a `SourceMember` object here. What do you think?

Reply:
Hmm we could, but only really care about the role for that specific squad.
What other props would you be interested in?

Reply:
Right now it's only this, but for future compatibility it's better to use the object.
We might want to present join date to the squad or notifications settings in the future and so on. Using the source member as an object here will give you support in all of them out of the box, whereas using only role may become legacy.